doc/user/inputs.rst

```diff
@@ -779,24 +779,26 @@ specifier
    hex dragger.
 
 xs types
-  The **cross-section type** is a single capital letter that identifies which cross section (XS) set
-  will be applied to this block. Each cross section set must be defined for at least one block with
-  fissile fuel. When the lattice physics code executes in ARMI, it determines the representative
-  blocks from each cross section type and burnup group and runs it to create the cross section set
-  for all blocks of the same type and in the same burnup group. Generally, it is best to set blocks
-  that have much different compositions to have separate cross section types. The tradeoff is that
-  the more XS types you define, the more CPU time the case will take to run.
+  The **cross-section type** is usually a single capital letter that identifies which cross section
+  (XS) set will be applied to the block. Each cross section set must be defined for at least one
+  block with fissile fuel. When the lattice physics code executes in ARMI, it determines the
+  representative blocks from each cross section type and burnup group and runs it to create the
+  cross section set for all blocks of the same type and in the same burnup group. Generally, it is
+  best to set blocks that have much different compositions to have separate cross section types. The
+  tradeoff is that the more XS types you define, the more CPU time the case will take to run.
+
+  Representing xsType by a single letter limits users to 26 groups. So ARMI will allow 2-letter
```

Comment:
You may or may not know this, but I am 99% certain that we allow for xsType to be numbers as well as letters. So with a-z and then 0-9, there are actually 36 possibilities.

We might want to update this type of language here as well as in the docstrings that I pointed out in the original issue (#1709 )

Reply:
Done!